pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -373,6 +373,13 @@ ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
 ARG SMD_DATA_PARALLEL_URL
 ARG SMD_MODEL_PARALLEL_URL
 
+# EFA checker binaries
+ARG EFA_CHECKER_DLC_SINGLE_NODE_URL=https://smdataparallel.s3.amazonaws.com/efa-health-checher/dlc-pytorch-2.0.1/efa_checker_single_node_dlc_with_mpi
+ARG EFA_CHECKER_DLC_MULTI_NODE_URL=https://smdataparallel.s3.amazonaws.com/efa-health-checher/dlc-pytorch-2.0.1/efa_checker_multi_node_dlc_with_mpi
+
+# Nvidia DCGM binary
```

Comment:
Same for this - Can we add a small line in the form of the comment that states - "Needed for SM Health Checker"?

Reply:
Added